indico/modules/events/editing/schemas.py

```diff
@@ -270,22 +271,30 @@ def _get_editable(self, contribution):
             return None
         return EditableBasicSchema().dump(editable)
 
+    def _get_persons(self, contribution):
+        return [person.full_name for person in contribution.person_links]
```

Comment:
look at `fields.Pluck` - no need for a custom method here, as long as there's a schema you can use for the person link (which is most likely the case)

Reply:
hmm, our `PersonLinkSchema` doesn't have the `full_name` attr, only the `display_full_name` which i'd rather not use since it might abbreviate it
there's also a `ContributionPersonLinkSchema` that's only used for JSON exports, maybe I can add the `full_name` attr there?